Combatant/tests/test_combatant.py

```diff
@@ -45,37 +44,48 @@
 )
 
 agent_path = os.path.join(
-    "/plark_ai_public", "data", "agents", "models", "latest"
+    "/plark_ai_public", "data", "agents", "models", "move_north_nn"
```

Comment:
could we use the AGENT_PATH from combatant.py here, so they are testing the agent that the combatant will use?